brage-import/src/test/java/no/sikt/nva/brage/migration/merger/CristinImportPublicationMergerTest.java

```diff
@@ -246,6 +252,51 @@ var record = new Record();
         assertThat(actualPublisher.getId(), is(equalTo(prioritizedPublisher.getId())));
     }
 
+    @Test
```

Comment:
Bør vi tenke på sequence number til contributors etter at vi har slått dem sammen?

Reply:
Lagde en test på akkurat dette, og i god TDD så forventet jeg at den ikke kom til å feile, noe den ikke gjorde. Men Rurik har lagd noe datamodellen som sørger for at hvis man legger på en liste med bidragsytere så blir sequence nummer oppdatert. Se her for implementasjonen: https://github.com/BIBSYSDEV/nva-datamodel-java/blob/6edea2935d32293de3026519dd3ae354f37c2a58/nva-datamodel-java/src/main/java/no/unit/nva/model/EntityDescription.java#L102C17-L102C32

Så derfor slettet jeg testen.